style.css

```diff
@@ -304,51 +298,73 @@ display: none;
   }
 
   .swap-class {
-    order: 4;
+    order: 1;
   }
-h1 {
- justify-content: end;
- 
-}
- 
-p {
-  width: 60%;
 
- }
+  p {
+    width: 80%;
+  }
 
- .connect {
-  width: 60%;
-  color: #7f8cff;
-  margin-bottom: 0;
-  text-decoration: none;
-}
+  .connect {
+    width: 60%;
+    color: #7f8cff;
+    margin-bottom: 0;
+    text-decoration: none;
+  }
 
-#fs-frm {
-max-width: 45%;
-margin: 0 auto;
+  #fs-frm {
+    max-width: 45%;
+    margin: 0 auto;
+  }
 
+  .header-img {
+    text-align: center;
+    border-top-left-radius: 120px;
+    padding-top: 2%;
+    background-position: center;
+  }
```

Comment:
- [ ] Hey, please, let's make some adjustments by adding the correct background image and reducing the width of the text so as for the text not to be too much apart from the inputs. I made a highlight in the images below so you can see how to adjust it. Let's fix this to improve our design as much as we can below:
- [ ] Let's remove the black border on the input to improve our design to match the Figma design as much as we can.
- [ ] Let's center the button to improve our design below:


*Figma design*
![Screenshot from 2022-12-16 17-22-04](https://user-images.githubusercontent.com/59397159/208143977-787c818b-8f70-41f1-95d8-29fb877615ec.png)

*your design*
![Screenshot from 2022-12-16 19-02-30](https://user-images.githubusercontent.com/59397159/208161239-40816f42-e257-4e62-b590-af0de459b347.png)